usagov/usagov-2021
# USAGOV-524-hide-pages-in-menu: Checking against page type instead of …

Checking against page type instead of menu weight to hide from sidenav


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-524

## Description

## Testing
1. Navigate to localhost/page-error. You should see "Contact us" highlighted in red on the side nav (it's currently set as current-item).
2. Create a new Page Type in taxonomy - "Standard Page- Nav Hidden" 
3. Navigate back to localhost/page-error and press edit. 
4. Select new page type from dropdown.
5. Refresh cache.
6. You should no longer see "Contact us" highlighted in red

## Checklist for the Developer



## Checklist for the Peer Reviewers
